bin/rename_fasta.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+'''Rename contigs of a FASTA file with incremental count.'''
+
+from sys import argv, exit
+import argparse
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('-i', '--input', help='Path to input FASTA file', required=True)
+    parser.add_argument('-p', '--prefix', help='FASTA header prefix before integer count', type=str, default='')
+    parser.add_argument('-o', '--output', help='Path to output FASTA file', required=True)
+    
+    if len(argv) < 2:
+        parser.print_help()
+        exit(1)
+    
+    return parser.parse_args()
+
+
+def main(input_filename, output_filename, header_prefix):
+    args = parse_args()
+    
+    logger.info('Processing {}...'.format(input_filename))
```

Comment:
I normally stay away from using `str.format` inside logger statements, as they are evaluated during execution even if the string is not printed. This is not really a problem here, but can lead to slowdowns if the logging statement is inside a hot loop. In my experience it's best to be consistent to avoid messing up when it really matters, so I normally just use e.g. `logger.info("string %s" % variable)`. I treat this as a special case for logging, as I normally prefer the new f-strings for formatting strings for output elsewhere (unless compatibility with older Python versions is required)